diff --git a/dependencies/pip/dev.txt b/dependencies/pip/dev.txt
index 4d153b37e..61f6705b2 100644
--- a/dependencies/pip/dev.txt
+++ b/dependencies/pip/dev.txt
@@ -236,7 +236,7 @@ prompt-toolkit==3.0.29
     # via
     #   click-repl
     #   ipython
-psycopg2==2.8.6
+psycopg2-binary==2.8.6
     # via -r dependencies/pip/requirements.in
 ptyprocess==0.7.0
     # via pexpect
@@ -293,8 +293,6 @@ requests-oauthlib==1.3.1
     # via msrest
 s3transfer==0.5.2
     # via boto3
-savreaderwriter @ https://bitbucket.org/fomcl/savreaderwriter/downloads/savReaderWriter-3.3.0.zip
-    # via -r dependencies/pip/requirements.in
 sentinels==1.0.0
     # via mongomock
 sentry-sdk==1.5.10
diff --git a/onadata/settings/base.py b/onadata/settings/base.py
index cf2d65df6..9f89ae29d 100644
--- a/onadata/settings/base.py
+++ b/onadata/settings/base.py
@@ -455,7 +455,7 @@ KOBOFORM_SERVER_PORT = env.str('KOBOFORM_SERVER_PORT', '8000')
 KOBOFORM_SERVER_PROTOCOL = env.str('KOBOFORM_SERVER_PROTOCOL', 'http')
 KOBOFORM_LOGIN_AUTOREDIRECT = True
 KOBOFORM_URL = env.url('KOBOFORM_URL', 'http://kf.kobo.local').geturl()
-KOBOFORM_INTERNAL_URL = env.url('KOBOFORM_INTERNAL_URL', KOBOFORM_URL).geturl()
+KOBOFORM_INTERNAL_URL = env.url('KOBOFORM_INTERNAL_URL', KOBOFORM_URL)
 KPI_HOOK_ENDPOINT_PATTERN = '/api/v2/assets/{asset_uid}/hook-signal/'
 
 # These 2 variables are needed to detect whether the ENKETO_PROTOCOL should overwritten or not.
